src/Common/src/TypeSystem/Common/InstantiatedType.cs

```diff
@@ -281,5 +281,21 @@ public override bool HasCustomAttribute(string attributeNamespace, string attrib
         {
             return _typeDef.HasCustomAttribute(attributeNamespace, attributeName);
         }
+
+        public override MetadataType GetNestedType(string name)
+        {
+            // It doesn't make sense to look at the types nested in an instantiated type in general.
+            // In particular, if the outer type is generic, and the inner type has an unrelated set of 
+            // generic parameters, the result is certain to be confusing.
+            throw new NotImplementedException();
```

Comment:
I was reading the GetNestedType definition from reflection and that is what they do:
> If a nested type is generic, this method returns its generic type definition. This is true even if the enclosing generic type is a closed constructed type.